lib/matplotlib/ticker.py

```diff
@@ -1918,13 +1948,14 @@ def _raw_ticks(self, vmin, vmax):
                 step = max(1, step)
             best_vmin = (_vmin // step) * step
 
-            low = np.round(Base(step).le(_vmin - best_vmin) / step)
-            high = np.round(Base(step).ge(_vmax - best_vmin) / step)
-            ticks = np.arange(low, high + 1) * step + best_vmin + offset
-            nticks = ((ticks <= vmax) & (ticks >= vmin)).sum()
+            edge = _Edge_integer(step, offset)
```

Comment:
Similarly, this whole block could use a couple lines of comment.  I looked at this the other day, and worked my way through it, but on its face it looks more complicated than whats needed.